app/models/course/assessment/question/text_response_comprehension_group.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+class Course::Assessment::Question::TextResponseComprehensionGroup < ApplicationRecord
+  self.table_name = 'course_assessment_question_text_response_compre_groups'
+
+  validate :validate_group_grade
+
+  has_many :points, class_name: Course::Assessment::Question::TextResponseComprehensionPoint.name,
+                    dependent: :destroy, foreign_key: :group_id, inverse_of: :group
+
+  accepts_nested_attributes_for :points, allow_destroy: true
+
+  belongs_to :question, class_name: Course::Assessment::Question::TextResponse.name,
```

Comment:
[Declare the associations together](https://github.com/Coursemology/coursemology2/wiki/Contributing-on-Rails#models) - can shift the rest as well?